README.md

```diff
@@ -80,17 +90,15 @@ Given that a user does not guess the word *house*,\
 When the has failed to guess the word within a sixty second time limit and/or have used up their total guesses,\
 Then they should be given zero points.
 
-
-
 ## Class Diagram
 
 ![dto](https://user-images.githubusercontent.com/37581557/151727563-0c1a89e0-cd50-4e15-bace-f62af457ec50.PNG)
 ![dao](https://user-images.githubusercontent.com/37581557/151719041-3ae7727f-0d31-4e7f-886b-f1ef2a286850.PNG)
 
-
 ## Json Schema
 
->{
+```json
```

Comment:
Using this syntax you can obtain syntax highlighting for the json schema

Reply:
Huh.  Neat.